azure-pipelines.yml

```diff
@@ -48,14 +48,28 @@ jobs:
       kind: micro
       architecture: x64
       pool: Hosted VS2017
-      queue: Windows.10.Amd64.ClientRS4.DevEx.15.8.Open
+      queue: Windows.10.Amd64.ClientRS4.Open
       csproj: src\benchmarks\micro\MicroBenchmarks.csproj
-      categories: ['coreclr', 'corefx']
+      runCategories: 'coreclr corefx' 
       frameworks: # for public jobs we want to make sure that the PRs don't break any of the supported frameworks
         - netcoreapp3.0
         - netcoreapp2.2
         - netcoreapp2.1
-        - net461
+
+ # Windows x64 micro benchmarks net461, public correctness job
```

Comment:
what's the reasoning for splitting this out?

Reply:
net461 needs to be on a VM with VS installed on it, which means it needs to be on a DevEx queue. I'm hoping to be able to move the netcoreapp legs to the Windows.10.Amd64.Open queue, so I split net461 out in preparation for that.

Reply:
cool. We may be able to take advantage of the new framework reference package but that can be another time.